### PR TITLE
New Rule: Oracle E-Business Suite Exploiation CVE-2022-21587

### PR DESCRIPTION
**Pull request by IsaAlMannaei**

### CODE REVIEW

File: rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: Oracle E-Business Suite Vulnerability
```
Comment (nasbench) [inline rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml:1]:
```suggestion
title: Potential CVE-2022-21587 Exploitation Attempt
```

File: rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+id: d033cb8a-8669-4a8e-a974-48d4185a8503
+status: experimental
+description: Critical arbitrary file upload vulnerability impacting Oracle E-Business Suite (EBS). The vulnerability also allows code execution. Multiple PoCs available.
```
Comment (nasbench) [inline rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml:4]:
```suggestion
description: Detects potential exploitation attempts of CVE-2022-21587 an arbitrary file upload vulnerability impacting Oracle E-Business Suite (EBS). CVE-2022-21587 can lead to unauthenticated remote code execution.
```

File: rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+references:
+    - https://www.rapid7.com/blog/post/2023/02/07/etr-cve-2022-21587-rapid7-observed-exploitation-of-oracle-e-business-suite-vulnerability/
+    - https://blog.viettelcybersecurity.com/cve-2022-21587-oracle-e-business-suite-unauth-rce/
+tags:
+    - attack.initial_access
+    - attack.t1190
+    - CVE-2022-21587
```
Comment (nasbench) [inline rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml:11]:
```suggestion
    - cve.2022.21587
```

File: rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+title: Oracle E-Business Suite Vulnerability
+id: d033cb8a-8669-4a8e-a974-48d4185a8503
+status: experimental
+description: Critical arbitrary file upload vulnerability impacting Oracle E-Business Suite (EBS). The vulnerability also allows code execution. Multiple PoCs available.
+references:
+    - https://www.rapid7.com/blog/post/2023/02/07/etr-cve-2022-21587-rapid7-observed-exploitation-of-oracle-e-business-suite-vulnerability/
```
Comment (nasbench) [inline rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml:6]:
```suggestion
    - https://www.rapid7.com/blog/post/2023/02/07/etr-cve-2022-21587-rapid7-observed-exploitation-of-oracle-e-business-suite-vulnerability/
    - https://attackerkb.com/topics/Bkij5kK1qK/cve-2022-21587/rapid7-analysis
```

File: rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+            -  '/OA_HTML/BneViewerXMLService?bne:uueupload=TRUE'
+            -  '/OA_HTML/BneUploaderService?bne:uueupload=TRUE'
+            -  '/OA_HTML/BneDownloadService?bne:uueupload=TRUE'
+            -  '/OA_HTML/BneOfflineLOVService?bne:uueupload=TRUE'
+    condition: selection
+fields:
+    - src_ip
+falsepositives:
+    - Vulnerability Scanners
+level: High
```
Comment (nasbench) [inline rules/web/webserver_generic/web_CVE_2022_21587_oracle.yml:29]:
```suggestion
level: high

```

**Review by nasbench — CHANGES_REQUESTED** [review]
Also please change the file name to `web_cve_2022_21587_oracle_ebs.yml` after applying the suggestions

**Review by nasbench — APPROVED** [review]

**Review by frack113 — APPROVED** [review]

**Pull request merged**